clusters/app.ci/cert-manager/registry-appci_certificate.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: Certificate
+metadata:
+  labels:
+    gcp-project: openshift-ci-infra
+  name: registry-appci
```

Comment:
I think the `appci` is implicit by the fact that this is on the `app,ci` cluster, maybe `registry-ci-openshift-org`?

Reply:
Updated.